www/js/con-author.js

```diff
@@ -178,10 +177,10 @@ define([
 			root_check.on('Change', lang.hitch(this, function(checked){
 					return this.disableHandlers || this.handleRoot(checked);
 			}));
-
-			var dynamic_check = registry.byId(this.controlMap.dynamic);
-			dynamic_check.on('Change', lang.hitch(this, function(checked){
-					return this.disableHandlers || this.handleDynamic(checked);
+			
```

Comment:
Please remove this empty space. Sorry for the finickiness of this review comment.